README.md

```diff
@@ -209,6 +209,21 @@ Note that the public leaderboard uses the test splits for all datasets except MS
 
 </details>
 
+
+<details>
```

Comment:
I thought we wanted this:  https://github.com/embeddings-benchmark/mteb/pull/1584#discussion_r1884403225

We might want to remove this section from the README.

Reply:
This was done according to the comment. I'll change the subset name to something different to make it less confusing

Reply:
I see, thanks.